UKHomeOffice/cop-ui
# COP-10221 - People component

# Description
This PR adds ta people component for the airpax target info page.

# To Test
- Pull & run against dev
- As the UI is unable to talk to the targeting-api, has testing can only be performed using mock data therefore comment out **`Line 119`** & uncomment **`Line 120`** in the file at the following path **`/client/src/pages/targets/airpax/TargetInfoPage.jsx`**.
- The People component should now be rendered.
